napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -126,8 +126,11 @@ def stop_annotating(self):
         self.record_annotations(self._annotation_model.get_curr_img_index())
 
         # Save rest of annotations, even if empty
-        for idx in range(self._annotation_model.get_curr_img_index()+1, self._annotation_model.get_num_images()):
-            self._annotation_model.add_annotation(self._annotation_model.get_image_at(idx), [])
+        self.record_annotations(self._annotation_model.get_curr_img_index())
+
+        for idx in range(self._annotation_model.get_num_images()):
+            if self._annotation_model.get_image_at(idx) not in self._annotation_model.get_annotations().keys():
```

Comment:
can just do `key in dict` comparison instead of calling `dict.keys()` method

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/d63044a1eff70697e1b1e08d7df117e71b5638bb